### DISCUSSION TITLE
probenecid PBPK model and conversion reference concentration v9- v11 PKsim

### DISCUSSION THREAD

**Discussion by paolamian01** [Open-Systems-Pharmacology/Forum#1544] [Questions & Problems]
Dear colleague,

I have a question about the probenecid PBPK model (PMID: 33237382, this model is also open access available on github https://github.com/Open-Systems-Pharmacology/Probenecid-Model) 

I wanted to convert the reference concentrations for both OAT3 and OATP1B1 from the probenecid PBPK model v9 in v11, but do not get the same concentrations in v9 and v11 after conversion. The concentrations are almost the same a lower time points in the concentration-time profile, but differences increases between v9 and v11 concentration with increasing time.

Is it possible that conversion of references concentrations for both OAT3 and OATP1B1 have already been made in the probenecid PBPK model which is upload on github?

Just to give some background on what I did:
- Downloaded the probenecid PBPK model from github and load this V9 probenecid model in V11, after which I exported one simulation to Mobi
- created identical probenecid simulation in V11, also exported this simulation to Mobi

-OAT3 V9 value in the kidney interstitial space is 0.04 umol. 
- OAT3 V11 value in the kidney interstitial space is 8.04E-3 umol 
- therefore the ratio is 0.8 
- within the expression profile block of OAT3 in V11 I have multiplied my reference concentration of 0.09 umol/L with 0.8 resulting in an " adapted"  reference concentration of 0.056 umol/L

For OATP1B1 the ratio is 0.667 in the periportal intersitital space in the liver. this value has been provided in the convertion tutorial op ESQlabs.
- the ratio of 0.667 has been multiplied with the reference concentration 0f 0.07 umol/L in V11 PK sim, resulting in an "adapted"  reference concentration 0f 0.04669 umol/L
-
- after comparing the PKsim simulation with the adapted reference concentration in V11 with those of V9 I got, for example, a concentration of 200.58 mg/mL in V11 and 201.19 mg/mL in V9 at t=1, while the difference is increased at, for example, t =12h (v11 79.81 mg/mL (V11) and 83.07 mg/mL (v9)

Does anyone have an idea about the differences who still exists between references concentrations in V9 and V11

Looking forward to the discussion

Paola

**Comment by tobiasK2001** [comment]
Dear Paola,

the model on Github you are referencing is still version 9. Currently we are working to provide an updated v11 model there.
But this is still work in the progress. Currently we are doing QC checks etc.
But good news: you can test our drfat-beta version! Please find the zip file for the snapshot attached
The only change to version 9  is that Kcat OAT3 is changed to this value: 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/37d62b83-4d55-4aff-a01b-75afc247858b)

using a correction factor 0.56897754 for OAT3

[Probenecid-Model_v11-beta.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/11573272/Probenecid-Model_v11-beta.zip)

Please let me know if you still see differences with this version.

Best, Tobias

**Reply by paolamian01** [reply]
Dear Tobias, thanks! this worked for OAT3

**Comment by StephanSchaller** [comment]
Dear Paola, there have been changes in how expressin data is loaded fromV9 to V10.

We have a short how-to, that @PavelBal or @sfrechen could upload (a PPT).

Best,
Stephan

**Reply by paolamian01** [reply]
Dear Stephan, thanks for your reply. It would be nice if the PPT can be uploaded as I still have some minor differences between V9 and V11 and want to be sure that I have the correct OATP1B1 correction factor as well.. maybe I have this from an old ppt presentation. best Paola
